src/test-helpers/delay.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Delay for tests.
+ */
+export const delay = (ms: number) => new Promise<boolean>(resolve => setTimeout(() => resolve(true), ms))
```

Comment:
You can pass `resolve` directly to `setTimeout` for a slightly more compact implementation, i.e. `setTimeout(resolve)`

Reply:
Will do that. Thanks!